src/perftest_parameters.c

```diff
@@ -807,8 +807,15 @@ static void change_conn_type(int *cptr, VerbType verb, const char *optarg)
 		fprintf(stderr," DC not detected in libibverbs\n");
 		exit(1);
 		#endif
+	} else if (strcmp(connStr[6], optarg) == 0) {
+		#ifdef HAVE_SRD
```

Comment:
as far as I understand, SRD works for SEND only. If so, add check for verb as it is done for UC and UD

Reply:
Right. Added check.